packages/app/ui/modal/ModalHeader.tsx

```diff
@@ -7,7 +7,7 @@ export function ModalHeader({ children }: PropsWithChildren) {
   return (
     <Dialog.Title
       as="div"
-      className="flex items-center w-full px-4 py-3 border-b border-surface-50"
+      className="flex items-center justify-between w-full px-4 pt-3"
```

Comment:
I the need to change theses styles?  

Related:
https://github.com/SwaprHQ/stackly-ui/pull/27/files#r1253526101

Reply:
Because as the `Modal` component uses now `divide-y` for style its children top borders, the bottom border styles from the ModalHeader clashed with the top border style (from `divide-y` in the Modal) from the ModalContent. This change was suggested by @berteotti